source/_docs/energy/water.markdown

```diff
@@ -27,3 +27,14 @@ Alternatively, the following shops sell ESPHome-based devices, that use a proxim
 Maybe you like to build one yourself? Pieter Brinkman has quite a [nice blog article on how to create your own water sensor](https://www.pieterbrinkman.com/2022/02/02/build-a-cheap-water-usage-sensor-using-esphome-home-assistant-and-a-proximity-sensor/) using ESPHome, or [build a water meter](https://www.ztatz.nl/p1-monitor-watermeter/) that works with the [P1 Monitor](/integrations/p1_monitor) integration.
 
 For any of the above-listed options, make sure it actually works with the type of water meter you have before getting one.
+
+## Manual Configuration
+
+Go to Settings > Dashboards > Energy and then click Add Water Source.
+
+If your water meter sensor isn't listed, be sure your sensor has the following parameters:
+- unit_of_measurement: "gal" or "m³" [the 3 must be superscript]
+- device_class: water
+- state_class: total_increasing
```

Comment:
This is the wrong way around. Once does not adjust a sensor to work with the energy dashboard. Instead, if a sensor provides the right data, it will work with the energy dashboard.

Reply:
That is very MQTT specific. When looking at the instruction provided:

> Go to Settings > Dashboards > Energy and then click Add Water Source.

> If your water meter sensor isn't listed, be sure your sensor has the following parameters:

For almost all integrations, there is no such thing as providing these parameters. In the end, when an integration (or in case you when defining templates/MQTT/protocol-based sensors), are responsible for providing as much and correct information as possible. It is not just this, but also for things like: icons 🤷 

The problem with adding this is that it isn't an end-user concern in almost all cases and, above all, isn't use-fixable. Adding statements like this will thus cause more issues then they resolve IMHO.